chicagobub/scripts/bubble_chart.js

```diff
@@ -123,7 +153,8 @@ function bubbleChart() {
     simulation.alpha(1).restart();
   }
 
-  function splitBubbles() {
+  function splitBubbles(year) {
+    yearToSplitOn = year;
```

Comment:
Here is the setting of that `yearToSplitOn`. The value comes from the now unique `id` values associated with the year buttons.